ReSharper.FSharp/src/FSharp/FSharp.Psi.Services/src/Util/FcsTypeUtil.fs

```diff
@@ -2,6 +2,12 @@
 
 open FSharp.Compiler.Symbols
 
+let rec getFunctionReturnType (fcsType: FSharpType) paramsToSkipCount =
```

Comment:
The function name is somewhat ambiguous. Consider `a -> b -> c`. What is the return type? Without the context I'd expect it to be `c`, but here we have another parameter that allows us to get `b -> c` instead. It's not good that one has to guess it from the parameters names instead of using the function name itself.

Can it be something like `skipFunctionParameters`?